hls4ml/model/profiling.py

```diff
@@ -4,8 +4,13 @@
 import numpy as np
 import pandas
 import seaborn as sb
+import uuid
+import os
+import shutil
+from collections import defaultdict
 
 from hls4ml.model.hls_model import HLSModel
+from hls4ml.converters import convert_from_config
```

Comment:
An unintended consequence of this seems to be that we now have a circular import, so when a user does `import hls4ml`, [this](https://github.com/fastmachinelearning/hls4ml/blob/master/hls4ml/model/__init__.py#L6) will fail. It's possible to `from hls4ml.model import profiling` afterwards, but we should see if there is some workaround to avoid an extra step.

Reply:
Do you have an example of the user code causing that circular import? I can import hls4ml just fine and tests here seem to pass as well.

Reply:
Right, I also see that `import hls4ml` works fine because of the `try: import ...` in `hls4ml/model/__init__.py`, but then I'm finding that using `hls4ml.model.profiling` will throw an error like `AttributeError: module 'hls4ml.model' has no attribute 'profiling'` because that `try` encountered an exception and `profiling` is not imported. If you temporarily move the `import` in that `__init__.py` outside of the `try`, you can see the cause.
How have you been using the `numerical` method after importing?
I find it works to do both:
```
import hls4ml
from hls4ml.model import profiling
```

Reply:
I see what you mean now. I remember encountering the same AttributeError and I had to write `import hls4ml.model.profiling` explicitly.

When I move the import out of the try-except block, I get a circular import error as well. I'll try to fix this.

Reply:
I've moved the problematic convert_from_config import to inside `get_unoptimized_hlsmodel()` function (which is the only place it's used) and this seems to solve the issue.

Reply:
Perfect, that fixes it